src/plugins/intel_gna/gna_plugin.cpp

```diff
@@ -661,6 +664,9 @@ void GNAPlugin::LoadNetwork(CNNNetwork & _network) {
         manager.register_pass<ngraph::pass::CommonOptimizations>();
         manager.register_pass<RemoveInputConvert>();
         manager.register_pass<RemoveOutputConvert>();
+        manager.register_pass<ngraph::pass::BidirectionalSequenceDecomposition>();
```

Comment:
It is already called in `CommonOptimizations`

Reply:
removed